h/templates/atom.xml

```diff
@@ -0,0 +1,37 @@
+{# Generic template for rendering an Atom feed given a feed dict.
+
+The feed dict should be a logical representation of an Atom feed as a Python
+dict, including a list of dicts for the feed's entries. This template will
+render the feed to Atom XML.
+
+#}<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You can make this a bit prettier by taking advantage of [whitespace control](http://jinja.pocoo.org/docs/dev/templates/#whitespace-control):

```jinja2
{#-

Blah blah blah

-#}
<?xml ...
```